CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ Versioned according to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+  * Adapt to feature selection/filtering mechanism for derived images in core
+  * Fixes for image feature-related corner cases in crop and deskew
```

Comment:
⛏ y: *image feature-related* -> *image-feature-related* (it is an adjective compound, https://www.grammarbook.com/punctuation/hyphens.asp)

Reply:
Inclined to disagree: English does not feature _Durchkopplung_ hyphenation, and `image feature` itself is a non-hyphenated compound noun. But here it would help prevent a garden path effect, so I should either hyphenate all or none.

Reply:
Pls. check the reference I added!

> Rule 1. Generally, hyphenate two or more words when they come before a noun they modify and act as a single idea. This is called a compound adjective.

Reply:
I did. But please bear in mind prescriptive grammar like this rarely accounts for the true semantics of the system, esp. in non-trivial cases. The formulation of _rule 1_ is somewhat imprecise (`single idea`), and I believe it is meant for set phrases (`state-of-the-art`, `matter-of-fact`) and multi-word lexemes (`dressed-up`, `burned-out`, `multi-word`) rather than recursive compounds. The only thing in that grammar which gets close is _rule 9a_, which states that proper nouns (`Academy Award`, `African American`, `Middle Eastern`) are _not_ to be hyphenated through.